lens/type.ml

```diff
@@ -1,13 +1,103 @@
+open Lens_utility
+
 type t = Lens of Sort.t
 
 let pp f _ = Format.fprintf f "Lens"
 
 let show _ = "Lens"
 
-let sort v =
-  match v with
-  | Lens sort -> sort
+let sort v = match v with Lens sort -> sort
```

Comment:
Is there a reason you can't / don't want to write `let sort (Lens sort) = sort` instead of doing the explicit match?

Reply:
Also, even though there's technically nothing wrong, using `sort` for both the function name and the contents of the Lens constructor is a little confusing...

Reply:
@SimonJF just personal preference, the other seems awkward to me, even if it is shorter.